data/templates/load-balancing/ipvsadm.rules.tmpl

```diff
@@ -0,0 +1,34 @@
+# Always clear the rules
+-C
+{% if rule is defined %}
+{%   for rule, config in rule.items() if config.disable is not defined %}
+{%       if config.protocol == 'tcp' %}
+{%         set protocol = '-t '   %}
+{%       elif config.protocol == 'udp' %}
+{%         set protocol = '-u '   %}
+{%       elif config.protocol == 'sctp' %}
+{%         set protocol = '--sctp-service '   %}
+{%       endif   %}
+{%     if config.virtual_ip | is_ipv4 %}
+-A {{ protocol }} {{ config.virtual_ip }}:{{ config.virtual_port }} -s {{ config.algorithm }}
+{%     else %}
+-A {{ protocol }} [{{ config.virtual_ip }}]:{{ config.virtual_port }} -s {{ config.algorithm }}
+{%     endif  %}
+{%     if config.backend is defined  %}
+{%         for backend, backend_config in config.backend.items() if config.backend.disable is not defined %}
+{%           if backend_config.backend_ip | is_ipv4 %}
+{%               set rule_cmd = '-a ' + protocol + config.virtual_ip + ':' + config.virtual_port + ' -r ' + backend_config.backend_ip + ':' + config.virtual_port  %}
```

Comment:
please use the `bracketize_ipv6` (https://github.com/vyos/vyos-1x/blob/current/python/vyos/template.py#L143) Jinja2 filter when dealing with IPv6 addresses that must be contained in `[]`

Reply:
done